src/api/todos.ts

```diff
@@ -17,4 +17,10 @@ export default [
     completed: false,
     userId: 4,
   },
+  {
+    id: 3,
+    title: 'quis ut nam facilis et officia qui',
```

Comment:
Consider using more descriptive variable names for better readability. For example, `title` could be `todoTitle` and `completed` could be `isCompleted`. 🤔